ironfish/src/sdk.ts

```diff
@@ -30,7 +30,6 @@ import { NodeUtils } from './utils'
 export class IronfishSdk {
   pkg: Package
   client: IronfishIpcClient
-  clientMemory: IronfishMemoryClient
```

Comment:
This has me wondering whether we need `client` to be a property of the sdk? 🤔

It seems like then we need to keep track of whether that client is connected. Do we have any use for a pre-connection client?